### PR TITLE
Add an integration test matching our quick start

### PR DESCRIPTION
**Pull request by mattdangerw**

### CODE REVIEW

File: keras_nlp/integration_tests/basic_usage_test.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import math
+
+import tensorflow as tf
+from tensorflow import keras
+
+import keras_nlp
+
+
+def test_basic_usage():
+    """This matches the quick start example in our documentation."""
```
[the remark this answers is not in the document]
Comment (mattdangerw) [reply keras_nlp/integration_tests/basic_usage_test.py:24]: No links live yet. I'll just say it's in our base README. I don't think we need to link a specific branch of the README.

File: keras_nlp/integration_tests/basic_usage_test.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import math
+
+import tensorflow as tf
+from tensorflow import keras
+
+import keras_nlp
+
+
+def test_basic_usage():
```
[the remark this answers is not in the document]
Comment (mattdangerw) [reply keras_nlp/integration_tests/basic_usage_test.py:23]: I'm not sure if it makes much difference here, but it seems good to use it for consistency. Switched.

File: keras_nlp/integration_tests/basic_usage_test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (chenmoneygithub) [reply keras_nlp/integration_tests/basic_usage_test.py]: sg

File: keras_nlp/integration_tests/basic_usage_test.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (chenmoneygithub) [reply keras_nlp/integration_tests/basic_usage_test.py]: thanks!

**Review by chenmoneygithub — APPROVED** [review]

**Pull request merged**